flocker/acceptance/test_api.py

```diff
@@ -369,6 +369,61 @@ def can_connect():
         d.addCallback(check_result)
         return d
 
+    def test_create_container_with_environment(self):
+        """
+        Create a container including environment variables on a single-node
+        cluster.
+        """
+        data = {
+            u"name": "my_env_container",
+            u"host": None,
+            u"image": "busybox:latest",
+            u"environment": {u"ACCEPTANCE_ENV_LABEL": 'acceptance test ok'}
+        }
+        waiting_for_cluster = wait_for_cluster(test_case=self, node_count=1)
+
+        def create_container(cluster, data):
+            data[u"host"] = cluster.nodes[0].address
+            return cluster.create_container(data)
+
+        d = waiting_for_cluster.addCallback(create_container, data)
+
+        def check_result((cluster, response)):
+            self.assertEqual(response, data)
+
+        def inspect_container():
+            try:
+                remote_data = run_SSH(
+                    22, 'root', data[u"host"],
+                    [
+                        b"docker",
+                        b"inspect",
+                        b"flocker--{0}".format(data[u"name"])
+                    ], None
+                )
+            except Exception:
+                return False
```

Comment:
This is a really bad pattern to use.  Don't ever handle `Exception` and then squash it down to a simple value like `False`.  Always handle the most specific exception type you can.  `Exception` is almost as far on the opposite end of this suggestion as you can get.  The exception that actually gets handled here could be anything from what you expected (what did you expect, by the way?) to a `NameError` because the function name you expected to call was misspelled to a `RuntimeError` because hardware failed and a random bit of memory got flipped.